src/open_clip/hf_model.py

```diff
@@ -14,6 +14,22 @@
     transformers = None
 
 
+# HF architecture dict:
+arch_dict = {
```

Comment:
for now keeping it in the same file since we're just getting it to work with RoBERTa but ideally we'd like to migrate this to a separate file